backend/src/db-utils.js

```diff
@@ -593,15 +599,18 @@ const queryAccountOutcomeTransactionsCount = async (accountId) => {
     // we must put 'lastDayCollectedTimestamp' as below to dislay correct value
     const timestamp =
       lastDayCollectedTimestamp ||
-      `(CAST(EXTRACT(EPOCH FROM DATE_TRUNC('day', NOW() - INTERVAL '1 day')) AS bigint) * 1000 * 1000 * 1000)`;
+      (new Date().getTime() - ONE_DAY_TIMESTAMP_MILISEC) * 1000 * 1000;
```

Comment:
I don't have hard preference at the moment, but I want to bring it to your attention that when you get the current time on JS side you start relying on the correct time being set on the server where node JS service is running while previously we relied on the time being properly set on the database side. This change should not bring any performance improvements since the timestamp value would be evaluated once on the database (compared with the case where you used it inside the `SELECT MAX(...)`)